src/rules/function-name-case/README.md

```diff
@@ -97,3 +97,125 @@ a {
   background: -WEBKIT-RADIAL-GRADIENT(red, green, blue);
 }
 ```
+
+## Optional options
+
+### `ignoreFunctions: ["/regex/", "non-regex"]`
+
+Ignore case of function names.
+
+For example, with `"lower"`, given:
+
+```js
+["SOME-FUNCTION", "/^get.*$/"]
+```
+
+The following patterns are considered warnings:
+
+```css
+a {
+  color: sOmE-FuNcTiOn();
+}
+```
+
+```css
+a {
+  color: OTHER-SOME-FUNCTION();
+}
+```
+
+```css
+a {
+  color: GetColor();
+}
+```
+
+```css
+a {
+  color: GET_COLOR();
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+a {
+  display: some-function();
```

Comment:
Sorry, I'm still a tad confused. Won't `[/FUNCTION-NAME/]` only allow `FUNCTION-NAME` and not `FuNcTiOn-NaMe`?